DependencyInjection/Compiler/CsrfExtensionPass.php

```diff
@@ -29,6 +29,17 @@ public function process(ContainerBuilder $container)
                 $tokenStorageReference = new Reference('security.context');
             }
             $definition->replaceArgument(0, $tokenStorageReference);
+
+            // Symfony <2.8 BC
+            $typeExtensionTag = $definition->getTag('form.type_extension');
+            if (method_exists($definition, 'setShared')) {
```

Comment:
you should detect the version of the Form component, not of the DI component (they may be different).

Use ``method_exists(AbstractType::class, 'getBlockPrefix')`` (without using the PHP 5.5 feature if needed for compat).